components/docker-up/docker-up/main.go

```diff
@@ -71,6 +72,8 @@ func main() {
 	pflag.BoolVarP(&opts.Verbose, "verbose", "v", false, "enables verbose logging")
 	pflag.BoolVar(&opts.RuncFacade, "runc-facade", true, "enables the runc-facade to handle rootless idiosyncrasies")
 	pflag.StringVar(&opts.BinDir, "bin-dir", filepath.Dir(self), "directory where runc-facade is found")
+	pflag.StringVar(&opts.ContainerIf, "iface", "ceth0", "container iface name")
```

Comment:
@kylos101 @aledbf We would instead rename `ceth0` to `eth0` in ws daemon that there is no different between regular and debug workspace. Would it be fine with you?